### DISCUSSION TITLE
Noi Ask empty

### DISCUSSION THREAD

**Discussion by Maxikozie** [lencx/Noi#310] [Q&A] [answered]
It might be a stupid question but how do I add the three windows back in Noi Ask? 
I had GPT, Gemini and Claude open. Now It says **Noi Ask - No Data**

Thx!
![image](https://github.com/user-attachments/assets/a382c8b9-20c1-42a4-8516-3fd9509f73b1)

**Comment by lencx** [comment, marked as answer]
You need to ensure that the [noi-ask](https://github.com/lencx/Noi/tree/main/extensions/noi-ask) plugin is up to date, then click the 🧠 icon on the sidebar to add or remove it.
<img width="246" alt="Screenshot 2024-12-24 at 01 08 37" src="https://github.com/user-attachments/assets/53de7e5e-242f-4550-bc8a-70bd6e4442fa" />

**Reply by Maxikozie** [reply]
thx for the answer, this worked!

**Comment by YYYYYangjijjia** [comment]
原来如此！
